hydra-filters/src/test/java/com/addthis/hydra/data/filter/bundle/TestBundleFilterTimeRange.java

```diff
@@ -13,121 +13,107 @@
-
-        String time = format.parseDateTime(input).getYear() + "" +
-                df.format(format.parseDateTime(input).getMonthOfYear()) + "" +
-                df.format(format.parseDateTime(input).getDayOfMonth()) + "" +
-                df.format(format.parseDateTime(input).getHourOfDay()) + "" +
-                df.format(format.parseDateTime(input).getMinuteOfHour());
-        assertEquals(input, time.substring(2));
+    public void testMinusBefore() throws IOException {
+        BundleFilterTimeRange filter = Configs.decodeObject(BundleFilterTimeRange.class, "time:TIME, before:-20160923, timeFormat:YYYYMMdd");
+        assertFalse(filter.filter(bundle));
     }
 
     @Test
-    public void testBundleFilterTimeRangeNoTimeZone() {
-        DateTimeZone tz = DateTimeZone.forID("EST");
-        DateTime ntzDateTime = utc.toDateTime(tz);
-        String input = ntzDateTime.toString("yyMMddHHmm");
-
-        BundleFilterTimeRange bftr = new BundleFilterTimeRange( timeIn, "121201", "130101", true, "yyMMddHHmm", null);
-        DateTimeFormatter format = bftr.getTimeZoneFormat();
+    public void testBeforeAfterInRange() throws IOException {
+        String str = "time:TIME, before:20170101, after:20120101, timeFormat:YYYYMMDD";
+        BundleFilterTimeRange filter = Configs.decodeObject(BundleFilterTimeRange.class, str);
+        filter.filter(bundle);
+        assertTrue( filter.filter(bundle));
+    }
 
-        String time = format.parseDateTime(input).getYear() + "" +
-                df.format(format.parseDateTime(input).getMonthOfYear()) + "" +
-                df.format(format.parseDateTime(input).getDayOfMonth()) + "" +
-                df.format(format.parseDateTime(input).getHourOfDay()) + "" +
-                df.format(format.parseDateTime(input).getMinuteOfHour());
-        assertEquals(input, time.substring(2));
+    @Test
+    public void testBeforeAfterOutRange() throws IOException {
+        String str = "time:TIME, before:1212011213, after:1301011415, defaultExit:true, " +
+                "timeFormat:yyMMddHHmm, timeZone=Australia/Brisbane";
+        BundleFilterTimeRange filter = Configs.decodeObject(BundleFilterTimeRange.class, str);
+        assertFalse(filter.filter(bundle));
     }
 
     @Test
-    public void testBundleFilterTimeRangeNoTimeFormat() {
-        DateTimeZone tz = DateTimeZone.forID("EST");
-        DateTime ntzntfDateTime = utc.toDateTime(tz);
-        String input = ntzntfDateTime.toString("yyMMddHHmm");
+    public void testNoTime() throws IOException {
+        String str = "time:null, before:20170101, after:20120101,timeFormat:YYYYMMDD";
+        try {
+            BundleFilterTimeRange filter = Configs.decodeObject(BundleFilterTimeRange.class, str);
+            fail("Expected exception: time field is required");
+        } catch(JsonMappingException jme) {
+            assertThat(jme.getMessage(), containsString("marked as required"));
```

Comment:
Verifying exception message here is OK, but the main verification is that an exception is thrown. Using `expected=JsonMappingException.class` is good enough and easier to read.